run_benchmark_multi.sh

```diff
@@ -16,15 +16,14 @@ hash=$(git rev-parse HEAD)
 make build-headless
 
 rm -rf "$work_dir/temp/"; mkdir "$work_dir/temp/"
-docker container stop garage_benchmark
-docker container rm garage_benchmark
+docker container stop garage_benchmark || true && docker rm garage_benchmark || true
```

Comment:
i think this can be achieved by adding `--rm` to `docker run`

Reply:
Would adding this cause the container to be deleted before the following `docker cp` command that copies the test results out?

Reply:
Probably. The better way to do this is to use the garage Makefile. This redirects the data outputs of the container onto the host filesystem.